src/components/CardArt.vue

```diff
@@ -74,12 +85,14 @@ $default-art-width: 626px;
   display: flex;
   align-items: center;
   justify-content: center;
+  text-align: center;
```

Comment:
both the style changes here are to make the text and x in error overlay be centered.